blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/JSONMetricsContainer.java

```diff
@@ -64,14 +65,18 @@ public JSONMetricsContainer(String tenantId, List<JSONMetric> metrics) {
         delayedMetrics = new ArrayList<Metric>();
         for (JSONMetric jsonMetric : jsonMetrics) {
             Locator locator;
+            boolean isValid;
```

Comment:
You were right, changed back to directly calling isValid().  It caused tests failure earlier for tenantId validation, but I think I was doing something else then that caused it to fail.  It's working now.